e2e/test_payloads_validator.py

```
@@ -0,0 +1,520 @@
+    # Encodings a detectar
+    ENCODING_MARKERS = ['BASE64:', 'HEX:', 'ROT13:', 'HTML:', 'UNICODE:']
+
+    @staticmethod
+    def extract_text_from_pdf(pdf_path: Path) -> str:
+        """Extrai todo o texto de um PDF"""
+        try:
+            reader = PdfReader(str(pdf_path))
+            text = ""
+            for page in reader.pages:
+                text += page.extract_text() + "\n"
+            return text
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)